viadot/sources/sap_rfc.py

```diff
@@ -0,0 +1,511 @@
+import re
+import logging
+from collections import OrderedDict
+from typing import List, Literal
+from typing import OrderedDict as OrderedDictType
+from typing import Tuple, Union
+
+import pandas as pd
+
+try:
+    import pyrfc
+    from pyrfc._exception import ABAPApplicationError
+except ModuleNotFoundError:
+    raise ImportError("pyfrc is required to use the SAPRFC source.")
+from sql_metadata import Parser
+
+from viadot.config import CONFIG
```

Comment:
This has changed in 2.0 (credential handling; see existing sources).

Reply:
Updated in the [Updated credentials to config.yaml](https://github.com/dyvenia/viadot/pull/582/commits/7b2dbffe3d8f2aaffee5b9cf72f7c8487413ab61)